source/loader/layers/validation/ur_valddi.cpp

```diff
@@ -8975,6 +8975,14 @@ __urdlllocal ur_result_t UR_APICALL urEnqueueTimestampRecordingExp(
             return UR_RESULT_ERROR_INVALID_NULL_POINTER;
         }
 
+        if (phEventWaitList == NULL && numEventsInWaitList > 0) {
```

Comment:
this is a generated file, you'll want to add these [here](https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/exp-enqueue-timestamp-recording.yml#L64) like [this](https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/enqueue.yml#L60)

Reply:
snap!